nodeup/pkg/model/networking/common.go

```diff
@@ -21,20 +21,25 @@ import (
 	"k8s.io/kops/upup/pkg/fi"
 )
 
-// WeaveBuilder installs weave
-type WeaveBuilder struct {
+// CommonBuilder runs common tasks
+type CommonBuilder struct {
 	*model.NodeupModelContext
 }
 
-var _ fi.ModelBuilder = &WeaveBuilder{}
+var _ fi.ModelBuilder = &CommonBuilder{}
```

Comment:
@johngmyers I just realised that the name was actually `networking.CommonBuilder`.
Should be better like this than `networking.NetworkingBuilder`.